src/Context.java

```diff
@@ -0,0 +1,11 @@
+public class Context {
+    private Command command;
+
+    public void setCommand(Command command){
+        this.command = command;
+    }
+
+    void exucuteCommand(UsersBook usersBook){
+        command.exucute(usersBook);
```

Comment:
is this code compilable ? I thought your command does not accept users book as parameter anymore

Reply:
I have already deleted this file (class) on my computer from the idea, I don't have it :)